docs/reference/nuspec.md

```diff
@@ -215,9 +240,15 @@ For example:
 </package>
 ```
 
+When uploading a package to nuget.org, the `type` attribute is limited to 100 characters and the `url` attribute is
+> limited to 4000 characters.
```

Comment:
It looks like the `>` here was added by accident. Same comment for line 250 and 351.

Reply:
See:

* https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-2128#repository
* https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-2128#title
* https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-2128#dependencies-element